coverage/src/BamView-WebComponent.js

```diff
@@ -165,8 +177,26 @@ class BamViewChart extends HTMLElement {
                 this.updateBamView();
             });
 
+            this.broker.addEventListener('stats-stream-data', (event) => {
+                const data = event.detail.coverage_hist;
+                let coverageMean = 0;
+                for (const coverage in data) {
+                    const freq = data[coverage];
+                    coverageMean += (coverage * freq);
+                }
+                this._meanCoverage = Math.floor(coverageMean);
+                this._bamView.updateMeanLineAndYaxis(this._meanCoverage);
+            });
+
             document.addEventListener('region-selected', (event) => this.handleGoClick(event.detail));
             document.addEventListener('gene-entered', (event) => this.handleSearchClick(event.detail));
+
+            document.addEventListener('brushed-region-change', (event) => this.handleRegionsInput(event));
+            document.addEventListener('selected-gene-change', (event) => this.handleGeneInput(event));
+            document.addEventListener('brushed-region-whole-genome', (event) => {
```

Comment:
Same issue as above. I'd call this event "global-brushed-region-change" or "brushed-region-global-change".